spec/openapi.yaml

```diff
@@ -97,6 +97,10 @@ tags:
       Locales are the language and country pairs that are used to identify the
       source content and the desired target locale for your translated content.
 
+  - name: Peoples
+    description: |-
+      People - it is user of the system who works, processes, translates context.
```

Comment:
All Smartling users, who process and translate content with the Smartling platform.